pkg/api/networkservice/mechanisms/vxlan/helpers.go

```diff
@@ -65,13 +66,6 @@ func (m *Mechanism) SetSrcIP(ip net.IP) *Mechanism {
 	if m == nil {
 		return nil
 	}
-	srcIP, ok := m.GetParameters()[SrcIP]
```

Comment:
I don't quite follow why this logic was removed.

Reply:
The main point to delegate managing of Src/DstOriginalIP  to nsmgr-proxy

Reply:
@edwarnicke Do you mind this direction?

Reply:
Hmm... so here's my gut feeling:

1.  Its fine to delegate that to nsmgr-proxy at this time.
2. Eventually we'd probably like some methods to support *proxies around this
3. I'm not sure what form those methods would take, we don't need to figure that out now, and its probable they won't look like what we are doing in this setter.

Net-net: I'm fine with this :)